scripting/include/retakes.inc

```diff
@@ -199,10 +199,10 @@ native void Retakes_GetPlayerInfo(int client,
                                   char primary[WEAPON_STRING_LENGTH],
                                   char secondary[WEAPON_STRING_LENGTH],
                                   char nades[NADE_STRING_LENGTH],
-                                  int& health,
-                                  int& armor,
-                                  bool& helmet,
-                                  bool& kit);
+                                  int &health,
+                                  int &armor,
+                                  bool &helmet,
```

Comment:
Sorry - I'd rather not include this change. I have a fairly strong preference for the & on the type.

Reply:
No problem. Ive returned it to its original position.